example/ios/Podfile

```
@@ -1,59 +1,40 @@
-            File.open(name, "w") { |file| file.puts replace }
-            STDOUT.flush
-        end
-    end
-    Dir[dir + '*/'].each(&method(:find_and_replace))
+
+  post_install do |installer|
+    # https://github.com/facebook/react-native/blob/main/packages/react-native/scripts/react_native_pods.rb#L197-L202
+    react_native_post_install(
+      installer,
+      config[:reactNativePath],
+      :mac_catalyst_enabled => false,
```

RuboCop (1.81.1)

[convention] Avoid comma after the last parameter of a method call.

(Style/TrailingCommaInArguments)